src/coreclr/tools/aot/ILCompiler.ReadyToRun/IL/ReadyToRunILProvider.cs

```diff
@@ -104,7 +155,11 @@ public override MethodIL GetMethodIL(MethodDesc method)
                         return result;
                 }
 
-                MethodIL methodIL = EcmaMethodIL.Create(ecmaMethod);
+                if (!_manifestModuleWrappedMethods.TryGetValue(ecmaMethod, out var methodIL))
+                {
+                    methodIL = EcmaMethodIL.Create(ecmaMethod);
```

Comment:
Should we call <code>CreateCrossModuleInlineableTokensForILBody</code> or add the created <code>methodIL</code> to the <code>_manifestModuleWrappedMethods</code> cache here?

Reply:
No, that would cause determinism bugs in the compiler. I'll add a comment.